Rollbar/Serialization/Json/JsonScrubber.cs

```diff
@@ -217,8 +217,39 @@ public static void ScrubJsonPath(JObject jsonData, string scrubPath, string scru
                 return;
             }
 
-            var jProperty = jsonData.SelectToken(scrubPath)?.Parent as JProperty;
-            jProperty?.Replace(new JProperty(jProperty.Name, scrubMask));
+            var jToken = jsonData.SelectToken(scrubPath);
+            if (jToken != null)
+            {
+                var jProperty = jToken.Parent as JProperty;
+                if (jProperty != null)
+                {
+                    jProperty.Replace(new JProperty(jProperty.Name, scrubMask));
+                    return;
+                }
+            }
+
+            //to deal with the possible dotted data element name we need to perform some acrobatics here:
+            int indxLimit = scrubPath.LastIndexOf('.');
```

Comment:
I'd prefer to either spell this out -- `indexLimit` -- or use a short name `iLimit`, and same below with "indx". Reason being, my gut feeling is that someone will read this code, hear "index" in their head, and then later write code that spells out "index" rather than use this shortened form, leading to maintenance frustration and possibly bugs.

Reply:
@bishopb , thanks! Agree. Fixed.